packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -63,12 +64,70 @@ void main() {
       textScaleFactor: 1.62,
       padding: const EdgeInsets.all(9.10938),
       viewInsets: const EdgeInsets.all(1.67262),
+      alwaysUse24HourFormat: true,
     );
     expect(copied.size, const Size(3.14, 2.72));
     expect(copied.devicePixelRatio, 1.41);
     expect(copied.textScaleFactor, 1.62);
     expect(copied.padding, const EdgeInsets.all(9.10938));
     expect(copied.viewInsets, const EdgeInsets.all(1.67262));
+    expect(copied.alwaysUse24HourFormat, true);
+  });
+
+  testWidgets('MediaQuery.removePadding removes specified padding', (WidgetTester tester) async {
+    const Size size = const Size(2.0, 4.0);
+    const double devicePixelRatio = 2.0;
+    const double textScaleFactor = 1.2;
+    const EdgeInsets padding = const EdgeInsets.only(
+      top: 1.0,
+      right: 2.0,
+      left: 3.0,
+      bottom: 4.0,
+    );
+    const EdgeInsets viewInsets = const EdgeInsets.only(
+      top: 5.0,
+      right: 6.0,
+      left: 7.0,
+      bottom: 8.0,
+    );
+    const bool alwaysUse24HourFormat = true;
```

Comment:
Doesn't seem to buy much over just...errr... `true` :)

Reply:
Done :P